pom.xml

```diff
@@ -1,13 +1,15 @@
-<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>com.noumenadigital.platform</groupId>
-    <artifactId>npl-cli</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
-    <name>${project.artifactId}</name>
-    <description>NPL CLI is a command-line tool designed to interact with the NPL Runtime and NOUMENA Cloud
-    </description>
+    <artifactId>npl-cli-parent</artifactId>
+    <version>2025.1.2</version>
```

Comment:
I think we want a comment here that this NEEDS to be kept in sync with the platform version